### PR TITLE
PP-23 Reusable input

### PR DESCRIPTION
**Pull request by vrudnitskaya**
### Description
Add reusable input with custom styles
### Screenshots
inputs

### CODE REVIEW

File: src/components/auth/Login.jsx [hunk 1]
```diff
@@ -1,7 +1,13 @@
+import FormInput from "../common/FormInput";
 const Login = () => {
   return (
     <>
       <h1>Login Form</h1>
+      {/* testing how the inputs looks like */}
+      <form>
+        <FormInput id="email" type="email" placeholder=" ">Email address</FormInput>
```
Comment (JoHeyJo) [inline src/components/auth/Login.jsx:8]: Is passing an empty string for `placeholder` for both inputs a style choice to center the place holder text in the input? Removing the place holder attribute still shows the desired text except it is integrated into the border. Either is fine.

File: src/components/common/FormInput.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  type,
+  name,
+  value,
+  onChange,
+  placeholder,
+  children,
+}) => {
+  return (
+    <>
+      <div class="relative w-full mb-4">
```
Comment (JoHeyJo) [inline src/components/common/FormInput.jsx:12]: className

**Review by JoHeyJo — APPROVED** [review]
mock up looks good so far.

File: src/components/auth/Login.jsx (hunk 1 above)
Comment (vrudnitskaya) [reply src/components/auth/Login.jsx:8]: Yes, without the placeholder attribute the label is into the border after first input focus.

**Review by ElenaByc — COMMENTED** [review]
I tried to use this element in the registration form:  
First, it stretches across the entire width of the page, is that supposed to be so?  
The font of the entered text is very, very light, is that normal? I can hardly see it.  

input

**Comment by vrudnitskaya** [comment]
> I tried to use this element in the registration form: First, it stretches across the entire width of the page, is that supposed to be so? The font color of the entered text is very, very light, is that normal? I can hardly see it.
> 
> input

input takes the width of its div/form

**Review by ElenaByc — APPROVED** [review]
Looks good to me

File: src/components/common/FormInput.jsx
Comment (Tanyaa-a) [inline src/components/common/FormInput.jsx:1]: Looks good

**Review by Tanyaa-a — APPROVED** [review]

**Pull request merged**